Library/Homebrew/software_spec.rb

```diff
@@ -504,14 +504,19 @@ def root_url(var = nil, specs = {})
     end
   end
 
-  sig { params(tag: Utils::Bottles::Tag).returns(T::Boolean) }
-  def compatible_locations?(tag: Utils::Bottles.tag)
+  sig { params(tag: Utils::Bottles::Tag).returns(String) }
```

Comment:
The type signature here is wrong for the return value and causing it fail in testing.  Sometimes it will be a `Symbol` (`:any`, `:any_skip_relocation`), but other times it will be a path (`/usr/local` etc.).  What's the right type signature that covers both of those?

Reply:
Perhaps you want `T.any(Symbol, Pathname)`?

Reply:
Testing this locally, it looks like the return values are either `Symbol` or `String`, so I guess that's what we have to use here.  Thanks for the pointer about using `T.any`!